class/person.rb

```diff
@@ -1,8 +1,13 @@
-class Person
+require_relative 'nameable'
+require_relative 'capitalize_decorator'
+require_relative 'trimmer_decorator'
+
+class Person < Nameable
   attr_accessor :name, :age
   attr_reader :id
 
   def initialize(age = nil, name = 'Unknown', parent_permission: true)
+    super
```

Comment:
- [ ] There seems to be an error when running your program that reads ` wrong number of arguments (given 3, expected 0) (ArgumentError)` which is coming from the way `super` is called at line `10` here, please take note that when calling super without any arguments it will take the arguments from the current `initialize` constructor but the parent class constructor in the `Nameable` class doesn't accept any arguments, so you should call it using `super()` instead.

Reply:
Thank you for all the clear instructions sir . I will keep in mind all of these and update my codes.